### ISSUE TITLE
Form groups in overlays are not programmatically associated with their group label [iOS]

### ISSUE THREAD

**Issue by willslone** [covid-alert-ny/covid-green-app#238]
The “Select your county”, “Gender identity”, “Age range”, “Race”, and “Ethnicity” overlays include form groups with an overall group label, but the group label is not announced upon focusing a control in the form group. For example, the “35-49” form control in the “Age range” overlays is simply announced as “35-49”.

![“35-49” form control focused by VoiceOver and announced as “35-49”](https://user-images.githubusercontent.com/4437084/92025072-01ef6980-ed2d-11ea-817d-8257c06d24df.png)

Use the accessibilityLabel property to include the group label in the accessible name of each control. Remove the group label from the reading order using the accessibilityElementsHidden, as it is redundant when included in the accessibilityLabel of each radio button. The controls also do not have convey their role and state to VoiceOver users. See issue #243. 

```javascript
<View accessibilityElementsHidden={true}>
  <Text>Choose your age range</Text>
</View>

<TouchableOpacity
  onPress={…}
  accessibilityLabel='35-49 ,Age Range'
  accessibilityRole='radio'
  accessibilityState={{selected: false}}
>
```

**Comment by floridemai** [comment]
@willslone Is it best to include "Age range" in the label for each radio control rather than having the modal announce that is that age range the user has to choose and then only the options are read. Please re-assign if that's the case and I'll update. Thank you.

**Comment by willslone** [comment]
@floridemai I would include the group label (e.g., “Age range”) in the label for each radio button. Once that is done, the group label can be removed from the reading order and instead of focusing the group label when the overlay is opened, the first control could be focused.

**Comment by floridemai** [comment]
@That's address in the next build. I left the "Choose your age range heading" and the label have only "18 to 29, radio-button, Age range". Hopefully that's correct.

**Comment by willslone** [comment]
@floridemai That should be fine.
